tests/ha/ctdb.pm

```diff
@@ -18,7 +18,7 @@ use utils qw(systemctl file_content_replace zypper_call);
 
 sub run {
     # Exit of this module if we are in a maintenance update not related to samba
-    return 1 if is_not_maintenance_update('samba');
+    return 1 if (is_not_maintenance_update('samba') && is_not_maintenance_update('ctdb'));
```

Comment:
Thanks. I wasn't sure, but I saw some `ctbd$something` packages in one of the repos so I decided to add it. I'll remove it since this is the confirmation I needed.

Reply:
And removed. Thanks!